pagopa/pn-frontend-e2e
# PN-9240-e-2-e-destinatario-recapiti-legali-inserimento-pec

## Short description
There is a missing step in test related to the **PN-9240** as you can see in the [confluence page](https://pagopa.atlassian.net/wiki/spaces/PN/pages/868778282/Mittente+-+Mapping+test-scenari)

## List of changes proposed in this pull request
- added new feature step: `When Nella pagina I Tuoi Recapiti si visualizza correttamente il riquadro relativo alla PEC`
- added related method to the new feature step


## How to test
Simply run `mvn clean test -Dcucumber.filter.tags=@TA_inserimentoPECPF` and verify if the test works fine.
